python_modules/dagster/dagster/_core/definitions/asset_condition_evaluation_context.py

```diff
@@ -21,15 +22,28 @@
     from .asset_daemon_context import AssetDaemonContext
 
 
+def root_property(fn: Callable[[Any], Any]) -> Callable[[Any], Any]:
```

Comment:
That conversation certainly happened (although I also can't find a record of it) -- my plan was to do some refactoring upstack as that concern doesn't really impact the user (and it'd be nice to get the core stuff merged sooner rather than later)

Reply:
Maybe worth updating the PR description? The way it's currently phrased, it sounds like it's largely focused on this change that will be undone later.